### PR TITLE
update to 308 and fix redirectUri info

### PR DESCRIPTION
**Pull request by jennyf19**

### CODE REVIEW

**Comment by jmprieur** [comment]
Don't we need to add a. WithRedirectUri as well?

File: README.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ If you want to use your own application coordinates, please follow these instruc
     - Select **Register** to create the application.
 1. On the app **Overview** page, find the **Application (client) ID** value and record it for later. You'll need it to configure the Visual Studio configuration file for this project.
 1. In the list of pages for the app, select **Authentication**.
-   - In the **Redirect URLs** | **Suggested Redirect URLs for public clients (mobile, desktop)** section, check **urn:ietf:wg:oauth:2.0:oob**
+   - In the **Redirect URLs** | **Suggested Redirect URLs for public clients (mobile, desktop)** section, check **https://login.microsoftonline.com/common/oauth2/nativeclient**
  1. Select **Save**.
```
Comment (jmprieur) [inline README.md]: I think it's not enough. We also need to add:

 ```CSharp
.WithRedirectUri("https://login.microsoftonline.com/common/oauth2/nativeclient")
 ```
 
when building the app. We don't even have a constant for it in MSAL.NET :(

**Comment by jennyf19** [comment]
I added it. Also tested it and it appears to be working.

---
In reply to: [488124706](https://github.com/Azure-Samples/active-directory-dotnet-desktop-msgraph-v2/pull/18#issuecomment-488124706) [](ancestors = 488124706)

File: README.md (hunk 1 above)
Comment (jennyf19) [reply README.md]: I added this, do you want something in the ReadMe as well?

---
In reply to: [280002140](https://github.com/Azure-Samples/active-directory-dotnet-desktop-msgraph-v2/pull/18#discussion_r280002140) [](ancestors = 280002140)

**Comment by jennyf19** [comment]
@jmprieur ready for re-review

**Pull request merged**